api/build.gradle

```diff
@@ -29,10 +29,6 @@ buildscript {
         HIBERNATE_VERSION = '6.5.3.Final'
         JACKSON_VERSION = '2.18.3'
         LIQUIBASE_VERSION = '4.31.1'
-        // 1.5.7+ results in a runtime error:
-        // Logging system failed to initialize using configuration from 'null'
-        //  java.lang.NoSuchMethodError: 'java.lang.Object ch.qos.logback.classic.LoggerContext.getConfigurationLock()'
-        LOGBACK_VERSION = '1.5.6'
```

Comment:
The new Spring Boot selects the latest version of Logback (1.5.18) automatically, and the problem described is resolved.